auth/store.go

```diff
@@ -114,6 +112,9 @@ type AuthStore interface {
 
 	// IsAdminPermitted checks admin permission of the user
 	IsAdminPermitted(username string) bool
+
+	// GenSimpleToken produces a simple random string
+	GenSimpleToken() (string, error)
```

Comment:
we can make a generic one. for simple_token, username will not be used. but we can also keep this simple for now.